jena-arq/testing/ARQ/JS/js-query-2.rq

```diff
@@ -0,0 +1,4 @@
+PREFIX js: <http://jena/apache.org/ARQ/jsFunction#>
```

Comment:
Ditto previous comment, is it really `http://jena/apache.org`, and not `jena.apache.org`?